unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Common/ScrollRectSensitivityHandler.cs

```diff
@@ -0,0 +1,75 @@
+using UnityEngine;
+using UnityEngine.UI;
+using MainScripts.DCL.WebPlugin;
+using System;
+
+/// <summary>
+/// Attaching this component to a scroll rect to apply the scroll sensitivity based on the os based stored sensitivities
+/// </summary>
+[RequireComponent(typeof(ScrollRect))]
+public class ScrollRectSensitivityHandler : MonoBehaviour
+{
+
+    private const float windowsSensitivityMultiplier = 15f;
+    private const float macSensitivityMultiplier = 4f;
+    private const float linuxSensitivityMultiplier = 3f;
+    private const float defaultSensitivityMultiplier = 3.5f;
+
+    private ScrollRect myScrollRect;
+    private float defaultSens;
+
+    void Awake()
+    {
+        myScrollRect = GetComponent<ScrollRect>();
+        defaultSens = myScrollRect.scrollSensitivity;
+        SetScrollRectSensitivity();
+    }
+
+    private void SetScrollRectSensitivity() 
+    {
+        myScrollRect.scrollSensitivity = defaultSens * GetScrollMultiplier();
+    }
+
+    private float GetScrollMultiplier() {
+        switch (GetCurrentOs())
+        {
+            case OperatingSystemFamily.Windows:
```

Comment:
have you tried with [platform scripting symbols](https://docs.unity3d.com/Manual/PlatformDependentCompilation.html), for example `UNITY_STANDALONE_WIN`, `UNITY_STANDALONE_LINUX`, `UNITY_STANDALONE_OSX` ?

Reply:
I'd use compiler conditionals as Nico said as well

Reply:
Ohh but we are getting the OS from the browser data, then I think this current way is the only way to deal with different OS speed since the compiler will always read `UNITY_WEBGL` =S

Reply:
Yeah I used OperatingSystemFamily to have a single method for both webgl and standalone ones due to UNITY_WEBGL always being returned no metter the OS in the browser case